staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -147,43 +147,43 @@ func (o *AuditOptions) Validate() []error {
 		}
 	} else {
 		// check webhook configuration
-		if err := validateBackendMode(pluginwebhook.PluginName, o.WebhookOptions.BatchOptions.Mode); err != nil {
-			allErrors = append(allErrors, err)
-		}
-		if err := validateBackendBatchConfig(pluginwebhook.PluginName, o.LogOptions.BatchOptions.BatchConfig); err != nil {
-			allErrors = append(allErrors, err)
+		if o.WebhookOptions.enabled() {
+			if err := validateBackendBatchOptions(pluginwebhook.PluginName, o.WebhookOptions.BatchOptions); err != nil {
```

Comment:
the pattern we have everywhere else is to pass `o.WebhookOptions` to the validate func and check for nil (and ConfigFile) locally. This style complicates the code by making complexity non-local.

Reply:
Done.